packages/medusa-file-local/package.json

```
@@ -33,6 +33,7 @@
   "gitHead": "81a7ff73d012fda722f6e9ef0bd9ba0232d37808",
   "keywords": [
     "medusa-plugin",
-    "medusa-plugin-file"
+    "medusa-plugin-file",
+    "medusa-plugin-storage",
```

Can you remove the extra comma at the end?

```suggestion
    "medusa-plugin-storage"
```